src/lexer/scan.ts

```diff
@@ -534,14 +534,20 @@ export function scanSingleToken(parser: ParserState, context: Context, state: Le
           break;
 
         case Token.CarriageReturn:
+        case Token.LineFeed:
           consumeLineBreak(parser);
```

Comment:
Regarding my comment earlier. If you have a 2nd func arg you can pass it as true for CR. That was the entire point with the bitmasks you removed :)

Reply:
There's no need to do anything different for CR or LF. The two token values can probably be coalesced into `Token.LineBreak`.

Reply:
See my final push; I've combined these two to `Token.LineBreak`.

Reply:
Seems good now. I'm going to merge this and ship  a new version